google/mozc
# Correct wrong readings of Kenshi Yonezu

## Description
A clear and concise description of this pull request.

The correct kana notation of Kenshi Yonezu (米津玄師) is "よねづけんし".

https://reissuerecords.net/diary/%E9%96%8B%E8%A8%AD/
https://x.com/hachi_08/status/468758483311947777
https://x.com/hachi_08/status/560086438172057600

Mozc's dictionary07.txt has a wrong reading "よねずけんし".
Google Japanese Input can convert from wrong readings "よねずげんし" or "よねづげんし" to the kanji notation.
All of them should be corrected.

## Issue IDs
Issue and/or discussion IDs related to this pull request.

Regression by #962

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: [e.g. Windows 11, macOS 13.1, etc]
 - Steps:
   1. Type "よねづげんし", "よねずげんし", or "よねづけんし"
   2. Confirm all of them are corrected to "よねづけんし"

![image](https://github.com/user-attachments/assets/a154d26e-0e8e-47df-9dec-e9ebe1644e95)
![image](https://github.com/user-attachments/assets/8274cf72-a2a6-4499-ad7d-8cdc071ff4f9)


## Additional context
Add any other context about the pull request here.
